### ISSUE TITLE
Converter "convert.png" looks very pixelated when zooming in

### ISSUE THREAD

**Issue by Quincunx271** [paceholder/nodeeditor#161]
Example:

![image](https://user-images.githubusercontent.com/7417022/39445697-c696624e-4c70-11e8-921c-5b6dc4ef80bf.png)

**convert.png** is not nearly as pixelated as this, but maybe the use of `QPixmap` has something to do with this.

My first thought is to look for SVG support in Qt, or maybe give a larger size for the `QIcon::pixmap` function.

**Comment by paceholder** [comment]
Yeah, I noticed that earlier. I think I should dynamically read the current scaling and prepare an appropriately downsampled icon from some high-resolution version.

> My first thought is to look for SVG support in Qt, or maybe give a larger size for the QIcon::pixmap function.

I tried both and still it looks weird. This code should scale the pixmap correctly according to the current scaling of the scene.

      QTransform t = painter->deviceTransform();

      QIcon icon(":convert.png");
      QPixmap pixmap = icon.pixmap(32 * t.m11(), 32 * t.m22());

![image](https://user-images.githubusercontent.com/322952/39619888-a1b218a8-4f89-11e8-80a8-69f98e868f49.png)

I think what we see is a properly scaled pixmap + additional stretching from the `QGraphicsView`. 

Maybe making a child `QGraphicsSvgItem` is the way to go.
